tests/com/github/bazelbuild/rules_jvm_external/resolver/MavenResolverTest.java

```diff
@@ -39,4 +42,19 @@ public void shouldSuccessfullyResolveNettyStaticClasses() {
     // There should be no cycle detected by this dependency
     resolver.resolve(prepareRequestFor(repo.toUri(), main));
   }
+
+  @Test
+  public void shouldErrorOnMissingTopLevelDependency() {
```

Comment:
This is probably something we want in `ResolverTestBase`, but we can land this PR and update.

Reply:
Since this one is checking for a Maven specific exception I put it in the MavenResolverTest as other resolvers would have a different exception.  We could just look for a RuntimeException and move it into `ResolverTestBase` if you want.